addons/project/models/mail_shortcode.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class MailShortcode(models.Model):
+    _inherit = 'mail.shortcode'
+    group_ids = fields.Many2many("res.groups", string="Authorized Groups", groups="base.group_system,project.group_project_manager")
+    user_ids = fields.Many2many("res.users", string="Authorized Users", groups="base.group_system,project.group_project_manager")
```

Comment:
(I am curious) Why do we allow the project manager to see/update those fields? 🤔 

In project, we don't really have a action/menu to allow to configure the canned response. 🤷🏼‍♂️ 

As your changes in `helpdesk` module, that changes could be completely erased by the ones in `im_livechat` module if `im_livechat` module is installed after that module.

Reply:
> (I am curious) Why do we allow the project manager to see/update those fields? 🤔

This is a question for @paju-odoo, I don't know exactly why. I didn't find any reason to challenge that.

> In project, we don't really have a action/menu to allow to configure the canned response. 🤷🏼‍♂️

It's inside the discuss app for everyone to access it. user_ids and group_ids are restricted config but everyone can create canned response for himself (only).

I m going to look into the group issue.

Reply:
> (I am curious) Why do we allow the project manager to see/update those fields? 🤔
> 
> In project, we don't really have a action/menu to allow to configure the canned response. 🤷🏼‍♂️
> 
> As your changes in `helpdesk` module, that changes could be completely erased by the ones in `im_livechat` module if `im_livechat` module is installed after that module.

The idea was to offer the feature to all managers, so that they could create canned responses for their whole team/ I can see that scenario happen in project. @xavierbol

Reply:
Hello @paju-odoo 

Okay, thanks for the explanation! 🙂